### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by viktor-yadelskyi**
test request

### CODE REVIEW

File: src/components/common/Button/Button.module.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@import 'src/styles/_variables';
+@import 'src/styles/_mixins';
+
+.content {
+  @include flex();
+  color: $super-white-color;
+  background-color: $btn-color;
+  border-radius: $border-radius;
+  font-size: 0.875rem;
+  height: 32px;
```
Comment (andrian-kars) [inline src/components/common/Button/Button.module.scss]: use 2rems here

File: src/components/common/Button/Button.module.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  padding: 0 1.5rem;
+  transition: 0.3s;
```
Comment (andrian-kars) [inline src/components/common/Button/Button.module.scss]: create a global var for transition

File: src/components/common/Button/Button.module.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  cursor: pointer;
+
+  //   @media (min-width: $tablet) {
```
Comment (andrian-kars) [inline src/components/common/Button/Button.module.scss]: What is this comment for?

File: src/components/common/Button/index.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import styles from './Button.module.scss';
```
Comment (andrian-kars) [inline src/components/common/Button/index.jsx]: use shortcut as s for styles

File: src/components/pages/ErrorPage/index.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import { Button } from '../../common';
```
Comment (andrian-kars) [inline src/components/pages/ErrorPage/index.jsx]: use alias as import from 'src/components/common'

File: src/components/pages/ErrorPage/index.jsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+import styles from './ErrorPage.module.scss';
+
+export const ErrorPage = ({ error }) => {
```
Comment (andrian-kars) [inline src/components/pages/ErrorPage/index.jsx]: change the props error name to errorCode, so it's more clearer

File: src/components/pages/ErrorPage/index.jsx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  let title, subTitle;
+  const navigate = useNavigate();
+
+  switch (error) {
+    case '400':
```
Comment (andrian-kars) [inline src/components/pages/ErrorPage/index.jsx:15]: for such errors constants should be created

File: src/components/pages/ErrorPage/index.jsx [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+      title = ' Invalid request.';
```
Comment (andrian-kars) [inline src/components/pages/ErrorPage/index.jsx]: What is this space for? For these strings should be localization used

File: src/components/pages/ErrorPage/index.jsx [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+      title = 'unknown error';
+      subTitle = 'an unknown error occurred';
+  }
+
+  return (
+    <div className={styles.errorPageContainer}>
+      <h1>{error}</h1>
+      <p>{title}</p>
+      <p>{subTitle}</p>
+      <Button label="button" handleClick={() => navigate(-1) || navigate('/')} />
```
Comment (andrian-kars) [inline src/components/pages/ErrorPage/index.jsx]: handler func needs to be created for such cases

File: src/App.jsx [hunk 10]
```diff
@@ -1,6 +1,6 @@
 import { Route, Routes } from 'react-router-dom';
```
Comment (andrian-kars) [inline src/App.jsx:1]: Name of commits should be changed

File: src/components/pages/ErrorPage/index.jsx (hunk 7 above)
Comment (viktor-yadelskyi) [reply src/components/pages/ErrorPage/index.jsx:15]: Where do I need to create the constants file?

**Pull request merged**